README.md

```diff
@@ -275,15 +275,15 @@ More extensive documentation for `daxctl` is available in the
 ## Testing CXL Memory
 
 After you run the [bootstrap script](#getting-started), the [benchmarks](benchmarks) subdirectory
-will contain several benchmark tools for testing CXL memory:
+will contain several micro-benchmark tools for testing CXL memory:
```

Comment:
I understand where this is coming from, but I didn't want to introduce the term "micro-benchmark" while using it interchangeably with "benchmark".

Reply:
Fair enough - I'll revert this line